Examples/Python/tests/test_edm4hep.py

```
@@ -0,0 +1,634 @@
+from acts.examples.odd import getOpenDataDetector, getOpenDataDetectorDirectory
+
+
+def assert_podio(
+    target_file: Path,
+    category: str,
+    collections: set[str] | None = None,
+    nevents: int | None = None,
+):
+    assert target_file.exists(), f"File {target_file} does not exist"
+    from podio.root_io import Reader
+    import cppyy
```

Ruff (0.8.2)

`cppyy` imported but unused

Remove unused import: `cppyy`

(F401)